src/tga_file.h

```diff
@@ -0,0 +1,60 @@
+
+
+#ifndef __TGA_FILE_H_INCLUDED__
+#define __TGA_FILE_H_INCLUDED__
+
+#include<iostream>
+#include<vector>
+#include<fstream>
+
+using namespace std;
+typedef char byte;
+
+class WriteTGA {
+public: 
+
+	byte header[18] = {
+		0,                  //1      id
+		0,                  //2      Color map type
+		0,                  //3      24bit uncompressed RGB
```

Comment:
Should be `2` for 24bit uncompressed RGB; see https://en.wikipedia.org/wiki/Truevision_TGA#Header > "Image type (field 3)".